packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-svg.tsx

```diff
@@ -126,40 +122,10 @@ export class ScDialSvg {
       .startAngle(endAngle2)
       .endAngle(endAngle1);
 
-    transition(this.stream?.id)
-      .duration(DURATION)
-      .ease(easeCubicInOut)
-      .attrTween('d', () => {
-        const endAngleColor = colorArc.endAngle()(ringD);
-        const startAngleColor = colorArc.startAngle()(ringD);
-
-        const endAngleDefault = defaultArc.endAngle()(ringD);
-        const startAngleDefault = defaultArc.startAngle()(ringD);
-
-        const interpolateColor = interpolate(
-          { startAngle: this.oldColorAngle.startAngle, endAngle: this.oldColorAngle.endAngle },
-          { startAngle: startAngleColor, endAngle: endAngleColor }
-        );
-
-        const interpolateDefault = interpolate(
-          { startAngle: this.oldDefaultAngle.startAngle, endAngle: this.oldDefaultAngle.endAngle },
-          { startAngle: startAngleColor, endAngle: endAngleColor }
-        );
-
-        this.oldColorAngle.startAngle = startAngleColor;
-        this.oldColorAngle.endAngle = endAngleColor;
-
-        this.oldDefaultAngle.startAngle = startAngleDefault;
-        this.oldDefaultAngle.endAngle = endAngleDefault;
-
-        return t => {
-          if (percent > 0) {
-            this.colorRing = colorArc.endAngle(interpolateColor(t).endAngle)(ringD) || '';
-          }
-          this.defaultRing = defaultArc.endAngle(interpolateDefault(t).endAngle)(ringD) || '';
-          return t;
-        };
-      });
+    if (percent > 0) {
+      this.colorRing = colorArc(ringD) || '';
+    }
```

Comment:
Can this be simplified?

```suggestion
    this.colorRing = percent > 0 ? colorArch(ringD) : ':';
```